rudder-core/src/main/scala/com/normation/rudder/services/policies/JavascriptEngine.scala

```diff
@@ -358,48 +354,54 @@ sealed trait JsEngine {
  */
 final object JsEngine {
   // Several evals: one default and one JS (in the future, we may have several language)
-  final val DEFAULT_EVAL = "$eval:"
-  final val EVALJS       = "$evaljs:"
+  final val DEFAULT_EVAL = "eval:"
+  final val EVALJS       = "evaljs:"
 
   final val PASSWORD_PREFIX       = "plain:"
-  final val DEFAULT_EVAL_PASSWORD = PASSWORD_PREFIX+"$eval:"
-  final val EVALJS_PASSWORD       = PASSWORD_PREFIX+"$evaljs:"
-  
+  final val DEFAULT_EVAL_PASSWORD = PASSWORD_PREFIX+DEFAULT_EVAL
+  final val EVALJS_PASSWORD       = PASSWORD_PREFIX+EVALJS
+
   // From a variable, returns the two string we should check at the beginning of the variable value
   // For a password, check if it's a plain text or not, otherwise check simply the eval keywords
   def getEvaluatorTuple(variable: Variable) : (String, String, Boolean) = {
     variable.spec.constraint.typeName match {
-        case t: AbstactPassword => (DEFAULT_EVAL_PASSWORD, EVALJS_PASSWORD, true) 
+        case t: AbstactPassword => (DEFAULT_EVAL_PASSWORD, EVALJS_PASSWORD, true)
         case _                  => (DEFAULT_EVAL, EVALJS, false)
     }
   }
-  
+
   final object DisabledEngine extends JsEngine {
     /*
      * Eval does nothing on variable without the EVAL keyword, and
      * fails on variable with the keyword.
      */
     def eval(variable: Variable, lib: JsRudderLibBinding): Box[Variable] = {
       val (default, js, _) = getEvaluatorTuple(variable)
-      variable.values.find( x => x.startsWith(default) || x.startsWith(js) ) match {
-        case None    => Full(variable)
+      variable.values.find( x => x.startsWith(default)) match {
```

Comment:
i think it's less readable to have two separated tests. Or maybe add a comment before (i guess it's for performance reason), and move the comme "Here, we nned to chose" just before this line